corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
:snowman:

@benrudolph I think you should still add it, so that we don't forget it's a paid feature once we remove the toggle.